README.md

```diff
@@ -70,6 +70,9 @@ return {
       '<CMD>lua require("activate").list_plugins()<CR>',
       desc = "Plugins",
     },
+  },
+  dependencies = {
+    { 'nvim-telescope/telescope.nvim', branch = '0.1.x', dependencies = { 'nvim-lua/plenary.nvim' } }
```

Comment:
This doesn't quite look correct, I think the second dependencies key is unnecessary?

Is specifying the branch also necessary?

Reply:
For telescope 'stable' yeah. And telescope needs plenary, so you need to add it as a dependency.

From the [telescope github](https://github.com/nvim-telescope/telescope.nvim):
```lua
{
    'nvim-telescope/telescope.nvim', tag = '0.1.4',
-- or                              , branch = '0.1.x',
    dependencies = { 'nvim-lua/plenary.nvim' }
}
```
I prefer using the branch method.

Reply:
Ok, the PR still needs updating to change this:

```  dependencies = {
    { 'nvim-telescope/telescope.nvim', branch = '0.1.x', dependencies = { 'nvim-lua/plenary.nvim' } }```

to this:

```
  dependencies = {
    { 'nvim-telescope/telescope.nvim', branch = '0.1.x' }, { 'nvim-lua/plenary.nvim' } }
```

Reply:
Yes, I was doing nested dependencies, though this isn't necessarily needed. I set edits to be allowed, so feel free to change this.